modules/administration-guide/partials/proc_configuring-bitbucket-server-oauth1.adoc

```diff
@@ -0,0 +1,118 @@
+..  Enter the application URL (see Application Link details page) and select **Create new link**.
+..  Select **Continue** on the warning message. This is not a problem.
+..  Complete the form:
+   - Application Name - Enter a name to help you identify this {prod-short} instance.
+   - Application Type - Leave as Generic Application.
+   - Service Provider Name - Enter the same name you used for Application Name.
+   - Consumer Key - Specify a consumer key. That is the content of `bitbucket_server_consumer_key` file.
+   - Shared secret - Specify shared secret. That is the content of `bitbucket_shared_secret` file.
+   - Request Token URL - `{your Bitbucket Server URL}/plugins/servlet/oauth/request-token`.
+   - Access token URL - `{your Bitbucket Server URL}/plugins/servlet/oauth/access-token`.
+   - Authorize URL - `{your Bitbucket Server URL}/plugins/servlet/oauth/access-token`.
+   - Create incoming link - Select this checkbox.
++
+[NOTE]
+====
+Bitbucket is not going to communicate with {prod-short}. There is no outgoing integration.
+It doesn't really matter what value is set as in `Request Token URL`, `Access token URL`, or `Authorize URL`.
+====
++
+image::bitbucket/bitbucket_link_applications.png[link="../_images/bitbucket/bitbucket_link_applications.png"]
+.. Select **Continue**.
+..  Complete the form:
+   - Consumer Key -  Specify a consumer key. That is the content of  `bitbucket_server_consumer_key` file.
+   - Consumer name - Enter the same name you used for Application Name.
+   - Public Key  - Provide the public key of your RSA key pair `public.pub`.
++
+[NOTE]
+====
+Value of public key should not include first `----BEGIN PUBLIC KEY-----` or last `-----END PUBLIC KEY-----` lines.
+[subs="+quotes,+attributes"]
+----
+cat public.pub | sed 's/-----BEGIN PUBLIC KEY-----//g' |  sed 's/-----END PUBLIC KEY-----//g' | tr -d '\n'
+----
+====
++
+image::bitbucket/bitbucket_link_applications_step2.png[link="../_images/bitbucket/bitbucket_link_applications_step2.png"]
+. Configure Bitbucket Server integration on {prod-short}
+.. Create a Kubernetes Secret in {prod-short} namespace
++
+[subs="+quotes,+attributes"]
+----
+$ {orch-cli} apply -f - <<EOF
+kind: Secret
+apiVersion: v1
+metadata:
+  name: github-oauth-credentials
+  namespace: <...> <1>
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: che-secret
+  annotations:
+    che.eclipse.org/mount-path: /home/user/eclipse-che/conf/oauth1/bitbucket
+    che.eclipse.org/mount-as: file
+data:
+  private.key: <...> <2>
+  consumer.key: <...> <3>
+  shared_secret: <...> <4>
+type: Opaque
+EOF
+----
+<1> {prod-short} namespace. The default is {prod-namespace}
+<2> base64 encoded content of `privatepkcs8.pem` without first and last lines.
+<3> base64 encoded content of `bitbucket_server_consumer_key` file.
+<4> base64 encoded content of `bitbucket_shared_secret` file.
+.. Configure {prod-short} server environment variables:
```

Comment:
maybe make sense to clarify that should be done by patching Che/CRW  CR? 
Or if it is done directly in config, restart of the server is required.